### PR TITLE
activity log: add "Why are Activity Logs required for support"

### PR DESCRIPTION
**Pull request by jstebbins**
What do you think?  Something we can point recalcitrant users to when they resist supplying a log.

### CODE REVIEW

**Comment by bradleysepos** [comment]
Thanks, some good information there. I can merge this in a few days when I'm back in town.

Do you think it should also be linked from the Table of Contents or just the main Activity Log article?

File: source/docs/en/latest/help/why-activity-log.markdown [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+reporting the version of HandBrake being used is prone to error. Activity Logs
+are definitive. Third, users often don't know the anwsers to questions we need
+to ask. And finally, Activity Logs help us improve the quality of HandBrake.
+Sometimes we will see problems in the log that you didn't notice.  When we
+see a problem, we follow up and fix it.
+
+## Can't I just tell you what you need to know without supplying an Activity Log?
+
+Perhaps.  And generally, if I see a support request that is missing an Activity
+Log, but has enough information to supply an answer, I will respond to that
```
Comment (sr55) [inline source/docs/en/latest/help/why-activity-log.markdown]: Use of We vs I. It's we above

File: source/docs/en/latest/help/why-activity-log.markdown [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+request.  On the other hand, if you don't supply enough information, I
+will not respond, not even to tell you what you are missing.  Dead air. There
+are other moderators that may respond depending on their mood and the phase
+of the moon. But you risk being totally ignored.
+
+## But what about my privacy?
+
+Maybe you are concerned about privacy and what the Activity Log might tell
+us and the world about you. I understand your concern. The vast majority
```
Comment (sr55) [inline source/docs/en/latest/help/why-activity-log.markdown]: We understand

File: source/docs/en/latest/help/why-activity-log.markdown [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+like file and path names in the log that you might prefer not sharing.
+There are a couple solutions. One, rename your source file before preparing
+the Activity Log you deliver to us. Two, edit the Activity Log to rename
+or remove file and path names you don't want to share.
+
+## What is in the activity log?
+
+The Activity Log is designed to answer a collection of questions that
+over years of supporting HandBrake users, we have found we frequently
+need answers to.  Here is a non-exhaustive list of such questions.
```
Comment (sr55) [inline source/docs/en/latest/help/why-activity-log.markdown]: : vs .

**Comment by jstebbins** [comment]
@bradleysepos, I'm not sure.  We should probably strive to keep the TOC from getting too big.  And I don't know if putting it more in peoples faces will actually help.  People seem to be very good at ignoring the red box at the top of the forum pages.  My thought when writing it was it would be a convenient link to supply whenever someone is resistant about supplying a log, instead of repeating the argument over and over.

File: source/docs/en/latest/help/why-activity-log.markdown (hunk 3 above)
Comment (jstebbins) [inline source/docs/en/latest/help/why-activity-log.markdown]: I don't understand this comment

File: source/docs/en/latest/help/why-activity-log.markdown (hunk 2 above)
Comment (jstebbins) [inline source/docs/en/latest/help/why-activity-log.markdown]: OK

File: source/docs/en/latest/help/why-activity-log.markdown (hunk 1 above)
Comment (jstebbins) [inline source/docs/en/latest/help/why-activity-log.markdown]: Ok.  I wasn't sure if you all agreed with my sentiment on this point.

**Comment by bradleysepos** [comment]
Taking a closer look at this, I think we could integrate it into the main Activity Log article while revising both. Care if I take a stab at it?

**Comment by jstebbins** [comment]
Fine by me.  Is it possible to link to a heading in the middle of an article?  I think the link I put in "Community Support" should take you directly to "Why are Activity Logs required for support?".

**Comment by bradleysepos** [comment]
I was thinking about putting that section at the top, but if that's not enough, can probably create an anchor for it manually. There are a few different ways to approach this, so let's see how my revision pans out first.

**Pull request closed without merging**